docs/mobile-apps/automated-testing/espresso-xcuitest/migration.md

```diff
@@ -0,0 +1,165 @@
+---
+id: migration
+title: Migrating to Espresso and XCUITest on Sauce
+sidebar_label: Legacy Migration
+description: Convert your TestObject Espresso and XCUItest setup to Sauce Labs testing.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+If you have been using Test Object to run your mobile automation tests on Espresso and XCUITest, use this guide to transition to Sauce Labs.
+
+Sauce Labs uses its framework agnostic test orchestrator [`saucectl`](/testrunner-toolkit) to execute tests according to the direction you specify in your configuration file.
+
+## What You'll Need
+
+* Sauce Labs account `USERNAME` and `ACCESS_KEY` [Look them up](https://app.saucelabs.com/user-settings)
+* Your Espresso or XCUITest apps and files
+
+## Installation & Setup
+
+`saucectl` can execute both Espresso and XCUITest tests, so the set up steps are the same regardless of which framework you are using.
+
+### 1. Install `saucectl`
+
+Begin by installing the `saucectl` CLI in the root directory of your existing mobile application automated testing project or your intended directory.
+
+```bash
+curl -L https://saucelabs.github.io/saucectl/install | bash
+```
+
+:::caution Required Minimum Versions
+Espresso requires `saucectl` version 0.36.0 or later and XCUITest requires `saucectl` version 0.44.0 or later.
+:::
+
+### 2. Check out the demo repositories
+
+Clone or download the [Espresso](https://github.com/saucelabs/saucectl-espresso-example) and [XCUITest](https://github.com/saucelabs/saucectl-xcuitest-example) example repos, as applicable to your project, to obtain the `saucectl` directory structure and example files for use as templates.
+
+### 3. Link your Sauce Labs account
+
+1. Generate a credentials file that `saucectl` can reference to authenticate your CLI commands.      
+    ```
+    saucectl configure
+    ```
+1. Enter your Sauce Labs `USERNAME` and `ACCESS_KEY` at the prompts.
+1. Add the resulting `credentials.yml` file in the `.sauce` folder of your project directory to your `.gitignore` file to protect it from unintended exposure.
```

Comment:
Same remark as previous one concerning that line.

Reply:
Thanks. This file is actually going away. It is a duplicate. I just haven't pulled it out yet.